app/uk/gov/hmrc/eoricommoncomponent/frontend/models/Service.scala

```diff
@@ -21,17 +21,23 @@ import play.api.Configuration
 import play.api.mvc.{PathBindable, Request}
 import uk.gov.hmrc.eoricommoncomponent.frontend.util.Constants
 
-case class Service(code: String, enrolmentKey: String)
+case class Service(code: String, enrolmentKey: String, callBack: String, friendlyName: String, friendlyNameWelsh: String)
 
 object Service {
 
+  val cds = Service("cds", "HMRC-CUS-ORG", "", "", "")
+  val empty = Service("", "", "", "", "")
+
   val configuration = Configuration(ConfigFactory.load())
 
   val supportedServices: Set[Service] =
```

Comment:
Suggestion for later.  I think we should store the supportedServices in a map keyed by code.
We also need to check when populating the map from config that there are no duplicate "codes" defined.  With current implementation one of the service definitions will get over-ridden (I think).

Reply:
yeah, good points